aws-blog-jobserver-emr/python_lambda/models.py

```diff
@@ -0,0 +1,98 @@
+import time
+import logging
+import json
+
+from sjsclient import client
+
+logging.basicConfig()
+logger = logging.getLogger()
+logger.setLevel(logging.INFO)
+
+#sjs = client.Client("http://ec2-54-172-7-213.compute-1.amazonaws.com:8090")
+sjs = client.Client("http://jobserver.ml-test-blog.internal:8090")
+myApp = sjs.apps.get("ml")
+myContext = sjs.contexts.get("ml-context")
+
+def testHandler(event, context):
+    #checkIncoming(event,context)
+    logger.info('got event{}'.format(event))
+    #conf = "{s3DataLoc:\"s3://dgraeberaws-blogs/ml/data/movielens/small/\",s3ModelLoc:\"s3://dgraeberaws-blogs/ml/models/movielens/recommendations/\"}"
+    #conf = '{'+'s3DataLoc:{},s3ModelLoc:{}'.format(s3DataLoc,s3ModelLoc)+'}'
+
+    s3DataLocProtocol = "\"s3://{}\"".format(event['s3DataLoc'])
+    s3ModelLocProtocol = "\"s3://{}\"".format(event['s3ModelLoc'])
+    conf = '{'+'s3DataLoc:{},s3ModelLoc:{}'.format(s3DataLocProtocol,s3ModelLocProtocol)+'}'
+
+    class_path = "com.amazonaws.proserv.ml.TestParams"
+    myJob = sjs.jobs.create(myApp, class_path, conf = conf, ctx = myContext)
+    myId = myJob.jobId
+    while myJob.status != "FINISHED":
```

Comment:
would be more readable and maintainable to define constants for hard coded values